docs/user-management/create-manage-apikeys.md

```diff
@@ -0,0 +1,21 @@
+---
+sidebar_position: 3
+title: Create and Manage API Keys
```

Comment:
API keys are not users. I don't think this belongs in User Management, probably either in Authentication or API Keys section.